src/bindings/Node.re

```diff
@@ -0,0 +1,320 @@
+[@bs.module "./fs-stub.js"]
+external thisProjectsEsyJson: string = "thisProjectsEsyJson";
```

Comment:
Are these bindings also part of the original project or are they taken from somewhere?

As mentioned earlier, we should clearly isolate vendored code

Reply:
These bindings are not taken from anywhere. If anything, they could be contributed to [bs-vscode](https://github.com/glennsl/bs-vscode) (or anywhere) since they are very useful

Reply:
I see. Things can stay as is for now. I just want to make sure we aren't accidentally maintaining forks of anything.

Reply:
Should we port the bindings to OCaml as well then?